api/config/module/feedback/v1/feedback.proto

```diff
@@ -20,14 +20,16 @@ message Survey {
   RatingOptions rating_options = 3 [ (validate.rules).message = {required : true} ];
 }
 
-message SurveyOrigin {
-  oneof type {
-    option (validate.required) = true;
+enum Origin {
+  // currently these are the supported placements for feedback
+  ORIGIN_UNSPECIFIED = 0;
+  HEADER = 1;
+  WIZARD = 2;
+}
```

Comment:
wanted to use the same enum defined here https://github.com/lyft/clutch/blob/main/api/feedback/v1/feedback.proto#L21-L26 but wasn't able to import it so I tried the method here https://stackoverflow.com/questions/8553147/is-it-possible-to-use-external-enums-outside-message-definitions-with-protocol which also didn't work. Example of what I tried for the latter approach: I had this import defined at the top of the file `import "/feedback/v1/enums.proto";` and then imported the enum like so `clutch.feedback.v1.<enum name>` but got undefined errors. am I missing something? Also, is it good practice to try to reuse the enum?

Reply:
this should work:
```proto
import "feedback/v1/feedback.proto";

message SurveyOrigin {
  clutch.feedback.v1.Origin origin = 1 [ (validate.rules).enum = {defined_only : true} ];
  Survey survey = 2 [ (validate.rules).message = {required : true} ];
}
```

Reply:
this is what I had tried originally but in trying it again, I realized that the field validation on the import was causing the issue. i removed the validation and now it works. confirmed tho even without the validation, the config parsing will value if we pass in an undefined enum.

this was the output of the pb.validate.go file and the errors
````
	if _, ok := feedbackv1.Origin_name[int32(m.GetOrigin())]; !ok {
		err := SurveyOriginValidationError{
			field:  "Origin",
			reason: "value must be one of the defined enum values",
		}
		if !all {
			return err
		}
		errors = append(errors, err)
	}
````

For this line `if _, ok := feedbackv1.Origin_name[int32(m.GetOrigin())]; !ok {` the errors were `assignment mismatch: 2 variables but 1 value` and `undefined: feedbackv1`

didn't come across a related issue when doing a quick search so perhaps something to do with our proto compile script https://github.com/lyft/clutch/blob/main/tools/compile-protos.sh.

Reply:
if feedbackv1 is undefined that means the go_package directive is probably incorrect in one of the proto files

Reply:
go_package imports for both follow our usual pattern - [api](https://github.com/lyft/clutch/blob/main/api/feedback/v1/feedback.proto#L5) and [config](https://github.com/lyft/clutch/blob/main/api/config/module/feedback/v1/feedback.proto#L5) but looks like the issue above stemmed from both having the same package name `feedbackv1`. renamed the config one, added back the validation on the fields, and thinks work now.